oak_attestation_verification/src/util.rs

```diff
@@ -85,15 +83,15 @@ pub fn equal_keys(public_key_a: &[u8], public_key_b: &[u8]) -> anyhow::Result<bo
 }
 
 /// Verifies the signature over the contents using the public key.
-pub fn verify_signature(
+pub fn verify_signature_raw(
     signature: &[u8],
     contents: &[u8],
     public_key: &[u8],
 ) -> anyhow::Result<()> {
-    let signature = Signature::from_der(signature).context("invalid ASN.1 signature")?;
+    let sig = Signature::from_der(signature).context("invalid ASN.1 signature")?;
```

Comment:
nit: Why did it shorten?

Reply:
In order not to shadow the function parameter. It is probably technically OK but seems careless.